### PR TITLE
Add feature support platform_fee

### PR DESCRIPTION
**Pull request by Pkittipat**

### CODE REVIEW

**Comment by jirawat-opn** [comment]
better add unit test also

File: operations/charge.go [hunk 1]
```diff
@@ -51,6 +51,12 @@ func (req *ListCharges) Describe() *internal.Description {
 //
 //	fmt.Println("created charge:", charge.ID)
 //
+
+type PlatformFee struct {
+	Percentage float64 `json:"percentage,omitempty"`
+	Fixed      string  `json:"fixed,omitempty"`
```
Comment (opn-takisawa) [inline operations/charge.go]: Is it ok that this type is `string` because the types of `amount` are used `int64` in other points.

Comment (Pkittipat) [reply operations/charge.go]: Good point, I'll fix this.

**Review by opn-takisawa — APPROVED** [review]

**Comment by Pkittipat** [comment]
> better add unit test also

I'll add it later it block from something I can't write test code.

**Review by ntsd — APPROVED** [review]

**Pull request merged**